### PR TITLE
GUACAMOLE-204: Implement CAS Authentication

### PR DESCRIPTION
**Pull request by necouchman**
Here's a pull request for the documentation for the CAS authentication module.

### CODE REVIEW

**Comment by necouchman** [comment]
Built the manual and the HTML output seems to be good.

File: src/chapters/cas-auth.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<chapter xml:id="cas-auth" xmlns="http://docbook.org/ns/docbook" version="5.0" xml:lang="en"
+    xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>CAS Authentication</title>
+    <indexterm>
+        <primary>CAS Authentication</primary>
+    </indexterm>
+    <para>CAS is an open-source Single Sign On (SSO) provider that allows multiple applications
+        and services to authenticate against it and brokes those authentication requests to a
```
Comment (mike-jumper) [inline src/chapters/cas-auth.xml]: "brokes"?

File: src/chapters/cas-auth.xml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        <para>The CAS authentication extension is available separately from the main
+                <filename>guacamole.war</filename>. The link for this and all other
+            officially-supported and compatible extensions for a particular version of Guacamole are
+            provided on the release notes for that version. You can find the release notes for
+            current versions of Guacamole here: <link
+                xlink:href="http://guacamole.incubator.apache.org/releases/"
+                >http://guacamole.incubator.apache.org/releases/</link>.</para>
+        <para>The CAS authentication extension is packaged as a <filename>.tar.gz</filename>
+            file containing only the extension itself,
+                <filename>guacamole-auth-cas-0.9.11-incubating.jar</filename>, which must
```
Comment (mike-jumper) [inline src/chapters/cas-auth.xml]: I believe this is 0.9.12 as of the latest rebase.

File: src/chapters/cas-auth.xml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                your system, please consult <xref linkend="configuring-guacamole"/> before
+                proceeding.</emphasis></para>
+        <para>To install the CAS authentication extension, you must:</para>
+        <procedure>
+            <step>
+                <para>Create the <filename>GUACAMOLE_HOME/extensions</filename> directory, if it
+                    does not already exist.</para>
+            </step>
+            <step>
+                <para>Copy <filename>guacamole-auth-cas-0.9.11-incubating.jar</filename> within
```
Comment (mike-jumper) [inline src/chapters/cas-auth.xml]: Same here - 0.9.12.

**Comment by mike-jumper** [comment]
Eep - GitHub also reports that this PR now has merge conflicts - best to rebase against latest master to make sure merge will be clean.

**Comment by necouchman** [comment]
Rebased to current master and updated version to 0.9.12, and fixed the spelling error.

**Pull request merged**